Plugin: 青年大学习提交(Web UI)

**插件名称：**

青年大学习提交(Web UI)

**插件功能：**

利用httpx库，向后台提交数据完成大学习，返回完成截图

**PyPI 项目名：**

teenstudy

**插件 import 包名：**

TeenStudy

**插件项目仓库/主页链接：**

https://github.com/ZM25XC/TeenStudy

**标签：**

[{"label":"青年大学习","color":"#e5ea52"},{"label":"httpx","color":"#52ea5f"},{"label":"Web UI","color":"#5295ea"}]

# 📃 商店发布检查结果

> Plugin: 青年大学习提交(Web UI)

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 teenstudy 的信息如下：
    name         : teenstudy                                                                
     version      : 0.1.0                                                                    
     description  : 基于nonebot2异步框架的青年大学自动提交插件基于nonebot2的青年大学习自动提交插件，用于自动完成大学习，在后台留下记录，返回完成截图 
    
    dependencies
     - amis-python &gt;=1.0.6,&lt;2.0.0
     - anti-useragent &gt;=1.0.10,&lt;2.0.0
     - beautifulsoup4 &gt;=4.11.2,&lt;5.0.0
     - bs4 &gt;=0.0.1,&lt;0.0.2
     - httpx &gt;=0.23.3,&lt;0.24.0
     - Jinja2 &gt;=3.1.2,&lt;4.0.0
     - lxml &gt;=4.9.2,&lt;5.0.0
     - nonebot-adapter-onebot &gt;=2.2.1,&lt;3.0.0
     - nonebot-plugin-apscheduler &gt;=0.2.0,&lt;0.3.0
     - nonebot2 &gt;=2.0.0rc3,&lt;3.0.0
     - Pillow &gt;=9.4.0,&lt;10.0.0
     - python-jose &gt;=3.3.0,&lt;4.0.0
     - tortoise-orm &gt;=0.19.3,&lt;0.20.0
     - ujson &gt;=5.7.0,&lt;6.0.0
     - uvicorn &gt;=0.20.0,&lt;0.21.0
插件 TeenStudy 加载出错：
03-01 07:32:26 [SUCCESS] nonebot | NoneBot is initializing...
03-01 07:32:26 [INFO] nonebot | Current Env: prod
03-01 07:32:27 [SUCCESS] nonebot | Succeeded to import &quot;nonebot_plugin_apscheduler&quot;
03-01 07:32:27 [ERROR] nonebot | Failed to import &quot;TeenStudy&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;TeenStudy&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/TeenStudy/__init__.py&quot;, line 4, in &lt;module&gt;
    from . import models, utils, web
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/TeenStudy/web/__init__.py&quot;, line 2, in &lt;module&gt;
    from fastapi import FastAPI
ModuleNotFoundError: No module named &#x27;fastapi&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: 青年大学习-#e5ea52, httpx-#52ea5f, Web UI-#5295ea。</li><li>✅ 项目 <a href="https://github.com/ZM25XC/TeenStudy">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/teenstudy/">teenstudy</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4560,5 +4560,28 @@
     "homepage": "https://github.com/techotaku39/nonebot-plugin-ncm-saying",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "TeenStudy",
+    "project_link": "teenstudy",
+    "name": "青年大学习提交(Web UI)",
+    "desc": "利用httpx库，向后台提交数据完成大学习，返回完成截图",
+    "author": "ZM25XC",
+    "homepage": "https://github.com/ZM25XC/TeenStudy",
+    "tags": [
+      {
+        "label": "青年大学习",
+        "color": "#e5ea52"
+      },
+      {
+        "label": "httpx",
+        "color": "#52ea5f"
+      },
+      {
+        "label": "Web UI",
+        "color": "#5295ea"
+      }
+    ],
+    "is_official": false
   }
 ]

